### DISCUSSION TITLE
Help with backup / restore

### DISCUSSION THREAD

**Discussion by Jaysingr** [YOURLS/YOURLS#3608] [Q&A] [answered]
I installed YOURLS using Softaculous as I am not a tech savvy person so explain me in simple terms please.

First I created backup using the cPanel "Backup Wizard" and deleted my entire webpage to check how to restore it. 

After installing YOURLS by Softaculous. I clicked on restore I am receiving this error

Error: (XID bj6rnb) The system failed to open the file “/home/DOMAIN/integration/links/WHMCS_clientarea_shopping_cart_domain_transfer.adminconfig” for writing because of an error: No such file or directory

Please help me restore my links.

Thanks.

**Comment by dgw** [comment, marked as answer]
This sounds like a cPanel question, which is out of scope for help from us.

I suggest contacting technical support at your hosting company.

**Reply by Jaysingr** [reply]
I solved it yes it is indeed a cPanel related Q&A

and is there any way to move admin page from domain.com/admin to something else like domain.com/abcde

I have found more related answer in issues and pull request and this is the [newest ](https://github.com/YOURLS/YOURLS/pull/2747) one I could find

**Reply by dgw** [reply]
You have answered your own follow-up question. https://github.com/YOURLS/YOURLS/pull/2747#issuecomment-689047797 contains a no-plugin approach to do what you ask. (`user/cache.php` was properly documented later, at https://yourls.org/docs/guide/extend/special-files#custom-always-on-plugin.)
